### PR TITLE
HW 8

### PR DESCRIPTION
**Pull request by AlinaZvarych**

### CODE REVIEW

**Comment by AntBelyi** [comment]
Where are all your merged tasks?
Screenshot_3

File: src/main/java/hw_8/Main.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        MyBrowser myBrowser = new MyBrowser();
+        MyFile myFile = new MyFile();
+
+        Lecture30.finishWork(myBrowser);
+        Lecture30.finishWork(myFile);
+
+        @FunctionalInterface
+        interface SessionCloser {
+            void closeSession(List<CanBeClosed> entities);
+        }
```
Comment (AntBelyi) [inline src/main/java/hw_8/Main.java]: This functional interface should be ia a separate file. Please follow the approach from our lection

Comment (AlinaZvarych) [reply src/main/java/hw_8/Main.java]: done

File: src/main/java/hw_8/Main.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+public class Main {
+    public static void main(String[] args) {
+        MyBrowser myBrowser = new MyBrowser();
+        MyFile myFile = new MyFile();
+
+        Lecture30.finishWork(myBrowser);
+        Lecture30.finishWork(myFile);
+
+        // Використання функціонального інтерфейсу з окремого файлу
```
Comment (AntBelyi) [inline src/main/java/hw_8/Main.java]: comment on English

**Review by AntBelyi — APPROVED** [review]

File: src/main/java/hw_8/Main.java (hunk 2 above)
Comment (AlinaZvarych) [reply src/main/java/hw_8/Main.java]: done

**Pull request merged**